src/seedu/addressbook/AddressBook.java

```diff
@@ -712,14 +713,18 @@ private static void createFileIfMissing(String filePath) {
 
         showToUser(String.format(MESSAGE_ERROR_MISSING_STORAGE_FILE, filePath));
 
-        try {
+        createStorageFile(filePath, storageFile);
+    }
+
+	private static void createStorageFile(String filePath, final File storageFile) {
+		try {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.